pcdsdevices/device_types.py

```diff
@@ -40,3 +40,4 @@
 from .timetool import Timetool, TimetoolWithNav
 from .valve import GateValve, Stopper
 from .wfs import WaveFrontSensorTarget
+from .mpodApalis import MPODApalisModule4Channel, MPODApalisModule8Channel, MPODApalisModule16Channel, MPODApalisModule24Channel
```

Comment:
can you put this up above between mpod and mps? This file is very alphabetical except for the new imports here